ProjFS.Mac/PrjFSKextTests/HandleOperationTests.mm

```diff
@@ -536,7 +572,11 @@ -(void) testWriteFileHydrated {
             _,
             _,
             _,
-            nullptr));
+            _,
+            nullptr,
+            _,
+            _,
+            _));
```

Comment:
Sigh, one disadvantage fo tests and having to do maintance like this when the signatures change.  Thanks for keeping i up @pmj.